src/javascript/index.js

```diff
@@ -7,7 +7,7 @@ export {Switch};
 // Takes care of finding Switches within the site code.
 export var init = function() {
     var x, _switch,
-        switches = document.querySelectorAll("[data-type='simple-switch']");
+        switches = document.querySelectorAll("[data-type='simple-switch']:not(._simple-switch-checkbox)");
```

Comment:
NIT: this line looks like it's > 80 characters, would you mind fixing it so that it's under that?